packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -7,16 +7,30 @@ import 'dart:convert';
 
 import 'package:meta/meta.dart';
 import 'package:stream_channel/stream_channel.dart';
-
-import 'package:test/src/backend/runtime.dart'; // ignore: implementation_imports
-import 'package:test/src/backend/suite_platform.dart'; // ignore: implementation_imports
-import 'package:test/src/runner/plugin/platform.dart'; // ignore: implementation_imports
-import 'package:test/src/runner/plugin/hack_register_platform.dart' as hack; // ignore: implementation_imports
+import 'package:stack_trace/stack_trace.dart';
+
+import 'package:test_api/src/backend/runtime.dart'; // ignore: implementation_imports
+import 'package:test_api/src/backend/suite_platform.dart'; // ignore: implementation_imports
+import 'package:test_api/src/runner/platform.dart'; // ignore: implementation_imports
```

Comment:
I don't think having these "runner" files within `test_api` is providing any value and is in fact quite confusing. My suggestion is that we create a a new `test_core` package which contains the very basic running functionality. `flutter_tools` can depend only on `test_core` which will reduce the overall transitive dependencies. `flutter_test` can continue depending only on `test_api`. This is how it's structured within Google3.

Reply:
That sounds like a good approach, we'll talk tomorrow